sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/controller/IThroughputController.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.throughputControl.controller;
+
+import com.azure.cosmos.implementation.RxDocumentServiceRequest;
+import reactor.core.publisher.Mono;
+
+public interface IThroughputController {
+    /**
+     * Decides whether the throughputController can handle the request.
+     * Different level throughput controller will have its own criteria.
+     *
+     * @param request
+     * @return
+     */
+    boolean canHandleRequest(RxDocumentServiceRequest request);
+
+    /**
+     * Close all the scheduled tasks and any other resources need to release.
+     * @return
+     */
+    Mono<Void> close();
```

Comment:
Again, why do we need a `Mono<Void>` here ?

Reply:
As discussed offline, will keep init and close as Mono for now.